examples/build.gradle

```diff
@@ -24,7 +24,53 @@ sourceSets.main.java.srcDirs = []
 sourceSets.test.java.srcDirs = [ ]
 sourceSets.test.scala.srcDir 'src/test/scala'
 
-dependencies {
+if (!(new File(rootDir, 'cluster/build.gradle').exists())) {
+  ext {
+    scalaBinaryVersion = '2.11'
+    scalaVersion = scalaBinaryVersion + '.8'
+    sparkVersion = '2.1.1'
+    snappyVersion = '1.1.0'
+    snappySparkVersion = '2.1.1.6'
+    sparkDistName = "spark-${sparkVersion}-bin-hadoop2.7"
+    sparkCurrentVersion = '2.3.2'
+    sparkCurrentDistName = "spark-${sparkCurrentVersion}-bin-hadoop2.7"
+    log4jVersion = '1.2.17'
+    slf4jVersion = '1.7.25'
+    junitVersion = '4.12'
+    sparkJobServerVersion = '0.6.2.9'
+  }
+
+  repositories {
+    mavenCentral()
+    maven { url 'https://dl.bintray.com/big-data/maven' }
+    maven { url "https://repo.spring.io/libs-release" }
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
+    maven { url 'http://repository.snappydata.io/repository/internal' }
+    maven { url 'http://repository.snappydata.io/repository/snapshots' }
+    maven { url 'https://app.camunda.com/nexus/content/repositories/public' }
+  }
+
+  dependencies {
+
+    compile 'org.scala-lang:scala-library:' + '2.11.8'
```

Comment:
Don't see these many dependencies in else block below. (log4j, slf4j, scala-library, scala-reflect, junit, snappy-core, spark-*). Doesn't snappy-cluster pull in those?